syntax = "proto3";

package common.v1;


message BaseResponse {
    int32 code = 1;
    string message = 2;
} 